app/helpers/hostgroups_helper.rb

```diff
@@ -19,4 +19,11 @@ def parent_hostgroups
       accessible_hostgroups - @hostgroup.descendants - [@hostgroup]
     end
   end
+
+  def hostgroups_actions_array(hostgroup)
```

Comment:
just a suggestion:
```suggestion
  def hostgroups_dropdown_actions(hostgroup)
```

Reply:
Yes, you are right, it is better. Changing it.